src/components/Data/Collections/Browse.vue

```diff
@@ -1,13 +1,135 @@
+  import CollectionDropdown from '../Collections/Dropdown'
+  import DocumentItem from '../Documents/DocumentItem'
+  import CrudlDocument from '../../Common/CrudlDocument'
+  import {
+    searchTerm,
+    rawFilter,
+    basicFilter,
+    paginationFrom,
+    paginationSize
+  } from '../../../vuex/modules/common/crudlDocument/getters'
+  import {formatFromQuickSearch, formatFromBasicSearch, formatSort, availableFilters} from '../../../services/filterFormat'
+
   export default {
-    name: 'CollectionBrowse'
+    name: 'CollectionBrowse',
+    directives: [
+      jQueryCollapsible
+    ],
+    data () {
+      return {
+        availableFilters,
+        selectedDocuments: [],
+        documents: [],
+        totalDocuments: 0
+      }
+    },
+    components: {
+      Headline,
+      CollectionDropdown,
+      DocumentItem,
+      CrudlDocument
+    },
+    computed: {
+      displayBulkDelete () {
+        return this.selectedDocuments.length > 0
+      },
+      allChecked () {
+        return this.selectedDocuments.length === this.documents.length
+      }
+    },
+    methods: {
+      createDocument () {
+        this.$router.go({name: 'DataCreateDocument'})
+      },
+      isChecked (id) {
+        return this.selectedDocuments.indexOf(id) > -1
+      },
+      toggleAll () {
+        if (this.allChecked) {
+          this.selectedDocuments = []
+          return
+        }
+
+        this.selectedDocuments = []
+        this.selectedDocuments = this.documents.map((document) => document.id)
+      },
+      toggleSelectDocuments (id) {
+        let index = this.selectedDocuments.indexOf(id)
+
+        if (index === -1) {
+          this.selectedDocuments.push(id)
+          return
+        }
+
+        this.selectedDocuments.splice(index, 1)
+      }
+    },
+    vuex: {
+      getters: {
+        searchTerm,
+        rawFilter,
+        basicFilter,
+        paginationFrom,
+        paginationSize
+      }
+    },
+    events: {
+      'toggle-all' () {
+        this.toggleAll()
+      }
+    },
+    route: {
+      data () {
+        let filters = {}
+        let sorting = []
+        let pagination = {
+          from: this.paginationFrom,
+          size: this.paginationSize
+        }
+
+        // Manage query quickSearch/basicSearch/rawSearch
+        if (this.searchTerm) {
+          filters = formatFromQuickSearch(this.searchTerm)
+        } else if (this.basicFilter) {
+          filters = formatFromBasicSearch(this.basicFilter)
+        } else if (this.rawFilter) {
+          filters = this.rawFilter
+          if (filters.sort) {
+            sorting = filters.sort
+          }
+        }
+
+        if (this.sorting) {
+          sorting = formatSort(this.sorting)
+        }
+
+        // Execute search with corresponding filters
+        performSearch(this.$route.params.collection, this.$route.params.index, filters, pagination, sorting).then(res => {
```

Comment:
be carefull about usage of `this.$route.params` please use `collection` & `index` from properties